t2.lic

```diff
@@ -10,6 +10,7 @@ custom_require.call(%w[common drinfomon common-travel equipmanager])
 class T2
   include DRC
   include DRCT
+  include DRCA
```

Comment:
```suggestion
```
Technically, the `include DRCA` isn't necessary because you call `release_cyclics` already qualified with `DRCA`.

In Ruby, the `include <module>` copies all the methods from the `<module>` to this class as if they had been defined in this class. So it's unnecessary bloat in the long term.

Reply:
would it be equally valid to change the release_cyclics line to read
```
release_cyclics unless %w[Commoner Barbarian Thief].include?(DRStats.guild)
```
 and keep the include?

Reply:
The `include`s scattered around the codebase was from when we decided terseness was better. So we used `include` everywhere and didn't explicitly reference modules.

Over time we decided that being explicit with where helpers came from helped new people acclimate to the codebase. No one has really taken the time to refactor the entire codebase, though.

Reply:
Ha, and @asechrest just said the same thing below. That'll teach me to read the entire thread before responding...